roles/ansible/tower/config-ansibletower/tasks/install.yml

```diff
@@ -56,3 +56,12 @@
 
 - name: "Download and Install the 'oc' client for OpenShift interactions"
   shell: curl {{ oc_client_download_url }} | tar -C /bin/ -xzf -
+
+- name: "Copy certs"
+  copy:
+    src: "{{ item }}"
+    dest: /etc/tower/
+  with_items: 
+    - "{{ tower_custom_ssl_certificate.key }}"
+    - "{{ tower_custom_ssl_certificate.cert }}"
```

Comment:
Need to add a `when` check as these aren't required parameters and hence the installation should still succeed if these aren't specified.